feedback/forms.py

```diff
@@ -53,3 +50,13 @@ class Meta:
             ),
         }
         formfield_callback = formfield
+
+    send_email_to_reporter = forms.TypedChoiceField(
+        widget=RadioSelect(
+            attrs={"class": "govuk-radios__input", "required": "True"},
+        ),
+        choices=(("Yes", _("Yes")), ("No", _("No"))),
+        coerce=lambda x: x == "Yes",
+        required=True,
+        label=_("Do you want us to email you a copy of this report?"),
```

Comment:
not sure about this copy - let me know if you have something better

Reply:
Perhaps - _Would you like us to email you a copy of this report?_